assets/schema_input.json

```
@@ -29,7 +29,12 @@
                         "maxLength": 0
                     }
                 ]
-            }
+            },
+            "condition": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Condition entry cannot contain spaces"
+            },
```

If it should remain remove comma
```suggestion
            }
```